src/controllers/utils/buildDocs.js

```diff
@@ -8,7 +8,7 @@ import omit from 'lodash/omit';
 import Versions from '../../shared/constants/Versions';
 import { wordSchema } from '../../models/Word';
 import { exampleSchema } from '../../models/Example';
-import Dialects from '../../shared/constants/Dialects';
+import Dialects from '../../shared/constants/Dialects.ts';
```

Comment:
we don't want to include extensions in our imports - this comment applies for the rest of your changes

Reply:
Fixed @ijemmao